ui/lobby/src/view/setup/components/ratingDifferenceSliders.ts

```diff
@@ -13,54 +13,71 @@ export const ratingDifferenceSliders = (ctrl: LobbyController) => {
   const currentRatingMin = isProvisional ? -500 : setupCtrl.ratingMin();
   const currentRatingMax = isProvisional ? 500 : setupCtrl.ratingMax();
 
-  return h(
-    `div.rating-range-config.optional-config${disabled}`,
-    {
-      attrs: isProvisional
-        ? { title: 'Your rating is still provisional, play some rated games to use the rating range.' }
-        : undefined,
-    },
-    [
-      i18n.site.ratingRange,
-      h('div.rating-range', [
-        h('input.range.rating-range__min', {
-          attrs: {
-            type: 'range',
-            min: '-500',
-            max: '0',
-            step: '50',
-            value: currentRatingMin,
-            disabled: isProvisional,
-          },
-          on: {
-            input: (e: Event) => {
-              const newVal = parseInt((e.target as HTMLInputElement).value);
-              if (newVal === 0 && setupCtrl.ratingMax() === 0) setupCtrl.ratingMax(50);
-              setupCtrl.ratingMin(newVal);
-            },
-          },
-        }),
+  const ratingLabels = () => {
+    if (site.blindMode) return [];
```

Comment:
This function is only referenced once; in a scope where we already know that `site.blindMode` is false.